static/js/lib/comments.js

```diff
@@ -41,3 +41,14 @@ export const findComment = (
 
   return null
 }
+
+const stripReplies = R.evolve({
+  replies: R.always([])
```

Comment:
Not sure if it's important here but this looks like it would return the same empty array instance for each `replies` instance worked on, which would mean code which calls `.push` would add items to multiple different arrays. I think if we replace with `() => []` it would return a different empty array instance each time.